doc/arkode/guide/source/Usage/LSRKStep/User_supplied.rst

```diff
@@ -0,0 +1,45 @@
+.. ----------------------------------------------------------------
+   Programmer(s): Daniel R. Reynolds @ SMU
+                  David J. Gardner @ LLNL
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2024, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _LSRKSTEP.Usage.UserSupplied:
+
+User-supplied functions
+=============================
+
+Currently, there is only one user-supplied function for LSRKStep (in addition to the :c:type:`ARKRhsFn` arguments that define the IVP).
+
+
+
+
+.. _LSRKStep.Usage.DomEig:
+
+The dominant eigenvalue estimation
+----------------------------------
+
+When running LSRKStep with either the RKC or RKL methods, the user must supply a dominant eigenvalue estimation function of type :c:type:`ARKDomEigFn`:
+
+.. c:type:: int (*ARKDomEigFn)(sunrealtype* t, N_Vector y, sunrealtype* lambdaR, sunrealtype* lambdaI, void* user_data)
```

Comment:
I am not sure how the Jacobian function is implemented; however, I anticipate that we should need a function in the type `ARKRhsFn`, which is already provided no matter what. When implemented internally, `ARKDomEigFn` type of a function shouldn't be requested anymore (depending on the nature of the implementation this might still be requested as an initial guess). The statement "the user must supply" seem too strong but it only implies for the current state. When we add this feature, we can modify this statement accordingly.